eng/test-job.yml

```diff
@@ -203,6 +203,30 @@ jobs:
           - jitstress2_tiered
           - zapdisable
           - tailcallstress
+        ${{ if eq(parameters.testGroup, 'outerloop-jitstressisas') }}:
+          scenarios:
+          - jitstressisas_incompletehwintrinsic
+          - jitstressisas_nohwintrinsic
+          - jitstressisas_nohwintrinsic_nosimd
+          - jitstressisas_nosimd
+          - ${{ if or(eq(parameters.archType, 'x64'), eq(parameters.archType, 'x86')) }}:
```

Comment:
You can simplify the condition to `in(parameters.archType, 'x64', 'x86')`

Reply:
I don't think we need arm32, musls, rhel.

Reply:
Done.